components/server/src/server.rs

```diff
@@ -1442,10 +1478,16 @@ where
         let store_path = self.store_path.clone();
         let snap_mgr = self.snap_mgr.clone().unwrap();
         let reserve_space = disk::get_disk_reserved_space();
-        if reserve_space == 0 {
+        let reserve_raft_space = disk::get_raft_disk_reserved_space();
+        if reserve_space == 0 && reserve_raft_space == 0 {
             info!("disk space checker not enabled");
             return;
         }
+        let raft_path = engines.raft.get_engine_path().to_string();
+        let separated_raft_mount_path =
```

Comment:
Shouldn't `set_raft_disk_reserved_space` if mount is the same.

Reply:
use the default value zero. raft_almost_full_threshold/raft_already_full_threshold are not used if separated_raft_mount_path is false.

Reply:
Then there's no need to check the mountpoint again here.

Reply:
``` Rust
                let used_size = if !separated_raft_mount_path {
                    snap_size + kv_size + raft_size + placeholder_size
                } else {
                    snap_size + kv_size + placeholder_size
                };
```
Calculate the used size may need this value.

Reply:
The point is these two things are equivalent. There's no need to use separated_raft_mount_path either. In fact, a more readable way to write is `let separated_raft_mount_path = reserve_raft_space != 0`.

Reply:
separated_raft_mount_path is the result of whether raft path and kv path are in same mount point; reserve_raft_space is get from the config toml, it can be set to zero by manual. If reserve_raft_space is set to zero, it means user do not like disk usage effected by raft disk(raft disk status alway `Normal`).

Reply:
reserve_raft_space works only when separated_raft_mount_path is true.